Gruntfile.coffee

```diff
@@ -360,6 +360,17 @@ module.exports = (grunt) ->
           dest: firefoxDevPath + 'data/lib'
         } ]
 
+      integration:
+        files: [ {
+          # Copy compiled Chrome App code, required for integration tests
+          expand: true, cwd: chromeAppDevPath
+          src: ['**', '!**/spec', '!**/*.md', '!**/*.ts']
```

Comment:
I think you want src to be `**/*` and the exclusion currently only excludes a file called spec, but not, for example, `foo.spec.js`... ?